setup.py

```diff
@@ -47,7 +47,7 @@
     'Flask-Gravatar~=0.0,>=0.4.2',
     'Flask-Login~=0.0,>=0.4.0',
     'Flask~=0.0,>=0.12.4',
-    'IDUtils~=0.0,>=0.2.4',
+    'IDUtils~=1.0,>=1.0.1',
```

Comment:
I *think* you will need to bump `inspire-dojson` first: https://github.com/inspirehep/inspire-dojson/blob/b79528106277aebc9f4f57fe5dc00a437975072c/setup.py#L40. But let's see what Travis says.

Reply:
done